contracts/src/token/WIP.sol

```diff
@@ -11,6 +11,10 @@ contract WIP is ERC20 {
     event Withdrawal(address indexed to, uint amount);
     /// @notice emitted when a transfer of IP fails
     error IPTransferFailed();
+    /// @notice emitted when an invalid transfer recipient is detected
+    error InvalidTransferReceiver();
```

Comment:
There is an EIP in last call to make standard ERC20 transfer errors
https://eips.ethereum.org/EIPS/eip-6093
```
error ERC20InvalidReceiver(address receiver);
error ERC20InvalidSender(address sender);
error ERC20InvalidSender(address sender);
...
```

Reply:
renamed the error according to the EIP.